2022q3/openstack-on-freebsd.adoc

```diff
@@ -0,0 +1,42 @@
+=== OpenStack on FreeBSD
+
+Links: +
+link:https://www.openstack.org/[OpenStack] URL: link:https://www.openstack.org/[] +
+link:https://github.com/starbops/osf[OpenStack on FreeBSD] URL: link:https://github.com/starbops/osf[] +
+
+Contact: Chih-Hsin Chang <starbops@hey.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+OpenStack is an open-source cloud operating system for different types of resources like virtual machines and bare metals.
+However, OpenStack only unofficially supports FreeBSD as a guest system.
```

Comment:
Meaning of "unofficially" unclear. "only supports FreeBSD as a guest, and that support isn't official"? Or is it "only unofficially" as in "no official support"? Either way, rhe next sentence about FreeBSD as a host sounds like a non-sequitur. Can you explain?

Reply:
I think the sentence is not accurate. According to the [docs page](https://docs.openstack.org/image-guide/obtain-images.html#freebsd-openbsd-and-netbsd), it is the FreeBSD guest image is unofficially maintained, not the support of OpenStack is unofficial. OpenStack only maintains a [support matrix regarding the hypervisors](https://wiki.openstack.org/wiki/HypervisorSupportMatrix). To make the intent of the paragraph clear, I'll just remove the line:

> However, OpenStack only unofficially supports FreeBSD as a guest system.

What do you think?

Reply:
Removing it works for me.